SQL/Archive/17.2/2017-06-13-addUserLanguagePreference.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE `language` (
```

Comment:
@jstirling91 Finally getting back to my pull requests - should be in the default schema now!